test/test_python_dispatch.py

```diff
@@ -284,6 +287,25 @@ def test_error_for_unsupported_ns_or_kind(self) -> None:
         with self.assertRaisesRegex(ValueError, "reserved namespace"):
             my_lib1 = Library("prim", "DEF")
 
+    def test_returning_symint(self) -> None:
```

Comment:
Although no real use case, can you add a test to register an operator that returns `MemoryFormat`?

Reply:
Do we have MemoryFormat defined in python? I tried but didn't find one.

Reply:
In `toIValue`, we still return int if the real_type is MemoryFormat:

```
    case TypeKind::MemoryFormatType: {
      if (THPMemoryFormat_Check(obj.ptr())) {
        auto memory_format = reinterpret_cast<THPMemoryFormat*>(obj.ptr());
        return static_cast<int8_t>(memory_format->memory_format);
      }
      // For backwards compatibility
      return py::cast<int64_t>(obj);
    }
```

so I don't know if we can setup a meaningful test for MemoryFormat.

Reply:
Ok that makes sense.